# Removes the call to `__android_log_print()` when targeting Android because we get missing symbol errors on Android otherwise.

--- a/core/logging.cpp
+++ b/core/logging.cpp
@@ -70,20 +70,20 @@ void al_print(LogLevel level, FILE *logfile, const char *fmt, ...)
      */
     std::wstring wstr{utf8_to_wstr(str)};
     OutputDebugStringW(wstr.c_str());
-#elif defined(__ANDROID__)
-    auto android_severity = [](LogLevel l) noexcept
-    {
-        switch(l)
-        {
-        case LogLevel::Trace: return ANDROID_LOG_DEBUG;
-        case LogLevel::Warning: return ANDROID_LOG_WARN;
-        case LogLevel::Error: return ANDROID_LOG_ERROR;
-        /* Should not happen. */
-        case LogLevel::Disable:
-            break;
-        }
-        return ANDROID_LOG_ERROR;
-    };
-    __android_log_print(android_severity(level), "openal", "%s", str);
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
 #endif
 }
